bcachefs/bcachefsmodule.c

```diff
@@ -197,6 +197,11 @@ static PyObject *PyBcachefs_iterator_next(PyBcachefs_iterator *self)
         Bcachefs_dirent dirent = Bcachefs_iter_make_dirent(fs, iter);
         return Py_BuildValue("KKIU#", dirent.parent_inode, dirent.inode, (uint32_t)dirent.type, dirent.name, dirent.name_len);
     }
+    else if (bch_val && iter->type == BTREE_ID_inodes)
```

Comment:
Maybe put that block before the `BTREE_ID_dirents` block since `BTREE_ID_inodes == 1` and `BTREE_ID_dirents == 2`